cmd/testdata/expect/TestSaveThenOverrideMetaAndTransformAndViz.json

```diff
@@ -4,21 +4,21 @@
     "author": {
       "id": "QmeL2mdVka1eahKENjehK6tBxkkpk5dNQ1qMcgWi7Hrb4B"
     },
-    "message": "meta:\n\tupdated title\nviz added\ntransform added\nbody:\n\tadded row /",
```

Comment:
It is awesome that we are fixing this. I always wondered why this showed up in cases where it clearly wasn't true, thanks for the fix!